### ISSUE TITLE
[Bug] Mic displayed on widget when Show voice search option is disabled

### ISSUE THREAD

**Issue by Diana-Rus** [mozilla-mobile/fenix#11819]
## Prerequisites
* Have search widget set on device's homescreen.
    1. Steps to reproduce

1. Open app and go to Menu/Settings/Search
1. Disable option "Show voice search" option.
    1. Expected behavior
The mic is not present at widget level.
    1. Actual behavior
The mic is present at widget level.
    1. Device information

* Android device: Sony Xperia Z5 (Android 7)
* Fenix version: Nightly 6/22

**Comment by mcarare** [comment]
AFAIK that setting is only for the search bar, not for the widget. cc @sblatz for confirmation.

**Comment by sblatz** [comment]
That's correct :) Having this option also affect the search widget seems reasonable, though. Pinging UX for their thoughts.

**Comment by AmyYLee** [comment]
Assigning to @brampitoyo

**Comment by brampitoyo** [comment]
@sblatz Correct! If “Show voice search” is disabled under Settings, the mic icon should not be shown in the search widget.

**Comment by AndiAJ** [comment]
Still reproducible on Firefox 87.0.0 Beta 3 using a OnePlus A3 (Android 6.0.1)

**Comment by AndiAJ** [comment]
Still reproducible on **`Firefox 88.0.0 Beta 1`** using a OnePlus A3 (Android 6.0.1)
